lib/grid.rb

```diff
@@ -0,0 +1,37 @@
+require_relative "./grid_creator"
+class Grid
+  include GridCreator
```

Comment:
- It's not clear from the tests which methods are part of Grid, and which from GridCreator